pipettes/firmware/motor_configurations.cpp

```diff
@@ -213,6 +228,11 @@ auto motor_configs::hardware_config_by_axis(TMC2160PipetteAxis which)
                      .active_setting = GPIO_PIN_SET},
                 // LED PIN C11, active setting low
                 .led = {},
+                .sync_in =
+                    {// NOLINTNEXTLINE(cppcoreguidelines-pro-type-cstyle-cast)
+                     .port = GPIOD,
```

Comment:
I made the `sync_in` pin D2 for the left gear motor, right gear motor, and linear motor. Is this what we want to do?